### ISSUE TITLE
MacOS before 10.14 not supported, probably affecting Win before 10

### ISSUE THREAD

**Issue by TwinFan** [TwinFan/LiveTraffic#40]
**Describe the bug**
On a MacOS 10.12 installation LiveTraffic can't be loaded with error `Symbol not found: __ZTISt18bad_variant_access`.

**To Reproduce**
Start X-Plane with LT on a MacOS 10.12 system.

**Expected behavior**
Starts successfully.

**Technical Info:**
 - OS: macOS: Version 10.12.6 (Build 16G1618)
 - X-Plane version: X-Plane 11.26r2 (build 112601 64-bit)
 - LiveTraffic version: 0.80 beta

**Log.txt**
```
dlerror:dlopen(/Users/admin1/X-Plane 11/Resources/plugins/LiveTraffic/64/mac.xpl, 6): Symbol not found: __ZTISt18bad_variant_access
  Referenced from: /Users/admin1/X-Plane 11/Resources/plugins/LiveTraffic/64/mac.xpl
  Expected in: /usr/lib/libc++.1.dylib
 in /Users/admin1/X-Plane 11/Resources/plugins/LiveTraffic/64/mac.xpl
```

**Comment by TwinFan** [comment]
Reason is usage of std::variant in DataRefs::dataRefDefinitionT. I introduced this during porting to Windows to avoid the there not supported Dot-notation for initializing unions. But after doing so I had to lift MacOS version in build settings to 10.14. (Underlying reason is not std::variant as such, but an exception used in there, which apparently makes a new runtime library necessary, which is not available in older MacOs versions and likely also not in Windows before v10.)

Implementation changed once more, std::variant removed. Instead the struct handles type of dataRef itself. Initialization is by use of two constructors, one for int dataRefs, one for float DataRefs.
